features/bootstrap/FeatureContext.php

```diff
@@ -741,10 +741,14 @@ public function install_wp( $subdir = '' ) {
 
 		$subdir = $this->replace_variables( $subdir );
 
+		// Disable WP Cron by default to avoid bogus HTTP requests in CLI context.
+		$env              = self::get_process_env_variables();
```

Comment:
`$env` doesn't seem to be used?

Reply:
Ah, right, left-over from something I tested (providing an override).